src/runtime/runtime_rp2040.go

```diff
@@ -32,6 +35,16 @@ func sleepTicks(d timeUnit) {
 	if d == 0 {
 		return
 	}
+
+	if hasScheduler {
```

Comment:
Given your comment above 
> (up to approx 72mins per RP2040 datasheet, 4.6.3. Alarms).

Should there not be a check here similar to `if hasScheduler && d < 72*minute` ?

Reply:
Agree.
And after that we probably want a follow-up ticket to not busy loop for periods higher than 72 mins. Like light sleeping in chunks of 70 mins.

Reply:
@soypat @ysoldak  - thanks for review.  I don't think we need to change anything here.  The sleep duration is slightly unsual when there's a scheduler - what it really means is 'there is currently no go routine that will be runnable for at least d period'.  It's safe to return early, and even desirable, if a interrupt handler has done something to make another go routine runnable (e.g. sent a message on a channel).

The max sleep from the datasheet is due to truncation because the RP2040 hardware has a smaller register for alarms.  All that means is the 'lightSleep' may return early if the scheduler asks to sleep for > 72mins.  This is safe and normal, since the scheduler will look for any go routines to run, and if none - sleep once again.

We could do `d = min(d, 72mins)` - but it's of no real benefit, makes the code a little more complex and just (potentially) increases the binary size by a few bytes.